mindmaps-graql/src/main/java/io/mindmaps/graql/internal/reasoner/predicate/AtomBase.java

```diff
@@ -96,8 +90,8 @@ public AtomBase(AtomBase a) {
 
     @Override
     public boolean isRuleResolvable() {
-        Type type = getParentQuery().getGraph().orElse(null).getType(getTypeId());
-        return type != null && !type.getRulesOfConclusion().isEmpty();
+        Type type = getType();
+        return type != null && !getType().getRulesOfConclusion().isEmpty();
```

Comment:
This null check of the type, is this because of the meta-ontology ? Everything has a type except the meta-ontology.